### ISSUE TITLE
Does not work for Inkscape appimage

### ISSUE THREAD

**Issue by MortalHappiness** [linuxmint/xapp-thumbnailers#9]
Linux mint: 21.1
xapp-appimage-thumbnailer: 1.0.1

---

I download [Inkscape appimage](https://inkscape.org/release/inkscape-1.2.2/) and find that the thumbnail does not show up.

![image](https://user-images.githubusercontent.com/47914085/213512496-d1f042fe-16fc-4990-9652-9fe5001977b9.png)

After running `./Inkscape-b0a8486-x86_64.AppImage --appimage-extract`, I get a folder called the `squashfs-root` and there is a png image in it. Hope this helps.

![image](https://user-images.githubusercontent.com/47914085/213513130-0879dffd-7021-4cc1-a244-26c6b468d183.png)

**Comment by mtwebster** [comment]
It works ok for me - does it work if you hit F5 (refresh)?

**Comment by MortalHappiness** [comment]
Yes. Here is the screencast. For comparison, I put another appimage besides it. I pressed F5 and only the ganache appimage shows a thumbnail.

https://user-images.githubusercontent.com/47914085/213518341-72ca22e8-0498-4f59-bb2e-36d16f81cae0.mp4

**Comment by clefebvre** [comment]
OK, it looks like it contains a .DirIcon, lemme check.
